test/runner.sh

```diff
@@ -29,14 +27,14 @@ fi
 trap cleanup EXIT
 
 echo " --> Enabling SSH in the container"
-docker exec -t -i $ID /etc/my_init.d/00_regen_ssh_host_keys.sh -f
-docker exec -t -i $ID rm /etc/service/sshd/down
-docker exec -t -i $ID sv start /etc/service/sshd
+docker exec $ID /etc/my_init.d/00_regen_ssh_host_keys.sh -f
+docker exec $ID rm /etc/service/sshd/down
+docker exec $ID sv start /etc/service/sshd
 sleep 1
 
 echo " --> Logging into container and running tests"
 cp image/services/sshd/keys/insecure_key /tmp/insecure_key
-chmod 600 /tmp/insecure_key
 sleep 1 # Give container some more time to start up.
-ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i /tmp/insecure_key -p $SSHPORT root@127.0.0.1 \
```

Comment:
I could not connect using localhost, only with the containers' IP. I wanted to remove duplication between these lines and the `ssh` target in Makefile. Apparently, I've "reverted" someone's contribution though: https://github.com/scrapinghub/baseimage-docker/commit/2f0e1ad0b86039c8c79422810e75b3f36cadb262#diff-2c0426df66e885ae0431057d1c516e62

Reply:
I'm actually more interested in line 38 and 39, what effect will result when insecure_key isn't copied into the container, or if it will have any effect on the tests.

It looks like there is a new error raised during the `make test`:
```
Warning: Permanently added '[127.0.0.1]:32768' (ECDSA) to the list of known hosts.
mesg: ttyname failed: Inappropriate ioctl for device
```

This is the only strange thing I found that looks wrong.  I wonder if it's related to removing `-t -i` setting.